omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+  (enabled by :term:`Write Data` privilege)
+- (possibly) changing the ownership of the newly created conainers and
+  contained result images to the users (enabled by :term:`Chown` privilege)
+- upload, edit and delete official scripts usable by all OMERO users
+  (enabled by :term:`Upload Scripts` privilege)
+
+Workflow 3: Analyst in clients:
+
+- OMERO.insight or Insight-ImageJ plugin: Analyst has to be a member of the
+  group where the data is. Can draw ROIs and extract analysis results from
+  the ROIs and data in any type of group. Can save ROIs except in Private
+  groups. Can upload official scripts in OMERO.insight (any group type,
+  Analyst does not have to be a member of
+  any particular group for script upload).
+
+- OMERO.web, OMERO.insight, Insight-ImageJ plugin: Analyst can adjust
+  rendering settings and save them, upload
+  attachments with results, annotate (for example tag, key-value pairs,
+  rating, commenting). All works except in Private groups when done on
+  images belonging to others. See :help:`rendering <managing-data#rendering>`,
+  :help:`annotating <managing-data#annotating>`,
+  :help:`attaching files <managing-data#attach>`,
+  :help:`attaching data <managing-data#attach>`.
+
+- CLI: Upload of official scripts (any group type,
+  see :doc:`/developers/scripts/user-guide` and below),
+  upload of attachments with results, annotating
+  (not in private group), creating containers, import of resulting images
+  into groups you are not a member of, and transferring ownership of these
+  containers (any group type), transferring ownership of objects
+  (annotation, ROI, uploaded attachment with results)::
+
+    # Upload an official script
+    $ bin/omero script upload --official /PATH/TO?YOUR_SCRIPT
+    # Login to the group the original data are in
+    $ bin/omero -g testgroup login
+    # Create new Dataset
+    $ bin/omero obj new Dataset name=new-dataset
+    # Import result images into the Dataset
+    $ bin/omero import -T Dataset:name:new-dataset /PATH/TO/RESULT/IMAGES
+    # Transfer the ownership of the Dataset and
+    # of the contained images to the user with ID:55
+    $ bin/omero chown 55 Dataset:112
+
+.. _Workflow 4:
+
+Workflow 4: Group and Data Organizer
+------------------------------------
+Group and Data Organizer is a role of someone tasked with creation of new
+users and groups in OMERO and allocating the users to appropriate groups
+(also able to change the users' information such as e-mail and change
+group permissions level). These tasks are facilitated by the privileges
+:term:`Create and Edit Groups`,
+:term:`Create and Edit Users` and :term:`Add Users to Groups`.
+
+Group and Data Organizer might also be tasked with dealing with data left
+after OMERO users which left the institution. Organizer can transfer of
+ownership of the data of the leaving person (facilitated by :term:`Chown`
+privilege) to another user. Possibly, the new owner of the data is not a
```

Comment:
In cases where the new owner of the data may not be a member of the data group, the Organiser may...